packages/@react-aria/dnd/test/dnd.test.js

```diff
@@ -2470,6 +2470,7 @@ describe('useDrag and useDrop', function () {
       await act(async () => {
         await tree.rerender(<Test showInput2 />);
       });
+      act(() => jest.runAllTimers());
```

Comment:
Were these timer changes still needed? Is firing a blur event somehow causing a timer to fire?

Reply:
And there were some additional tests failing, specifically DialogTrigger `input in nested popover should be interactive with a click` and `will not lose focus to body`

Reply:
Those weren't actually needed. Looks like nested cases for DialogTrigger are failing